polling_stations/apps/file_uploads/templates/file_uploads/upload.html

```diff
@@ -27,61 +27,70 @@ <h2>Javascript disabled</h2>
     </noscript>
 
     <h1>{{ council.name }}</h1>
-    <h2>Upload EMS export</h2>
-    <p>
-        Use the buttons below to select and upload a polling station export file from your computer.
-        Instructions on how to create the export can be found <a href="https://democracyclub.org.uk/projects/polling-stations/upload/">here</a>.
-    </p>
-    <p>
-        In the event that some of the stations in your council area are managed by neighbouring councils for the GE,
-        please ensure that they are aware that they need to provide this data to us.
-    </p>
-
-    <form id="file_upload_form" method="POST">
-        {% if SHOW_DATE_PICKER %}
+    {%if NO_UPCOMING_ELECTIONS %}
+        <p>
+            We don't know about any upcoming elections administered by {{ council.name }}.
+            If you have data to upload, please <a href="mailto:pollingstations@democracyclub.org.uk.">get in touch</a>
+            to tell us which election you want to upload data for.
+        </p>
```

Comment:
Given we use this same block here

https://github.com/DemocracyClub/UK-Polling-Stations/blob/30e60a50939058f43ae57ff0154b0ec8dbb54fe1/polling_stations/apps/file_uploads/templates/file_uploads/council_detail.html#L88-L92

we could pull this out into a shared include.

https://docs.djangoproject.com/en/4.2/ref/templates/builtins/#include
You can pass variables in using `{% include .. with .. %}` to deal with the fact that the `council` object doesn't have the same name in both places.

Reply:
done in d630b7ebef6677f53d5a1fcf9ef1e0c2583e2f14